docs/tutorial/index.html

```diff
@@ -17,6 +17,194 @@
 <header id="title-block-header">
 <h1 class="title">Testo tutorial</h1>
 </header>
-<p>Coming soon!</p>
+<h2 id="introduction">Introduction</h2>
+<p>Testo is a test framework for OCaml. Like with OUnit or Alcotest, the user writes a collection of tests. A test consists of a name and an OCaml test function to run, with some options. If the test function returns, the test is considered successful but if it raises an exception, it is considered failed.</p>
+<p>The test suite is compiled into a test executable with a fancy command-line interface provided by the Testo library.</p>
+<h2 id="main-features">Main features</h2>
+<ul>
+<li>Most tests can be defined by providing only a name and a function of type <code>unit -&gt; unit</code>.</li>
+<li>Tests can be assigned nested categories of arbitrary depth but are conveniently handled as a single flat list.</li>
```

Comment:
You don't really understand the point of having nested categories here; maybe explain why nested category can be useful (improve output, can be use when selecting test to run all test in a category, etc.)